packages/labs/signals/README.md

```diff
@@ -183,19 +234,36 @@ export class SignalExample extends SignalWatcher(LitElement) {
   render() {
     return html`
       <p>The count is ${count}</p>
-      <button @click=${this._onClick}>Increment<button></button></button>
+      <button @click=${this.#onClick}>Increment<button></button></button>
     `;
   }
 
-  private _onClick() {
+  #onClick() {
     count.set(count.get() + 1);
   }
 }
 ```
 
+#### `withWatch()`
+
 `withWatch()` is a function that wraps an `html` tag function with the
-auto-watching functionality. This allows you to compose this wrapper with other
-html-tag wrappers like Lit's `withStatic()` static template wrapper.
+auto-watching functionality. The `html` tag exported by `@lit-labs/signals` is
+simply the core lit-html template tag wrapped with `withWatch()` as a
```

Comment:
Arthur once told me to try to not use words like “simply” or “just”. Maybe something like:

> The `html` tag exported by `@lit-labs/signals` is a convenient export of the core lit-html template tag wrapped with `withWatch()`.